core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -119,15 +119,16 @@ private[spark] class SparkArithmeticException(
     errorClass: String,
     errorSubClass: Option[String] = None,
     messageParameters: Array[String],
-    queryContext: String = "")
+    context: Option[QueryContextSummary] = None)
```

Comment:
IIUC `QueryContextSummary` is only a container of `QueryContext` and the summary string, which is not end-user facing but only an internal thing.

Reply:
I removed QueryContextSummary in the last commit.